src/serac/physics/utilities/functional/detail/metaprogramming.hpp

```diff
@@ -164,6 +175,9 @@ void for_loop(lambda f)
   }
 }
 
+/**
+ * @brief overload
```

Comment:
We should use the overload doxygen command:

https://www.doxygen.nl/manual/commands.html#cmdoverload

Reply:
Yep, this was a typo.

Reply:
I figured after I saw you use it below but was too tired to go back and edit my comment. Thanks for fixing it.